vagrant/scripts/enable_debug.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+touch /tmp/chroma-agent-debug
```

Comment:
This should have a provisioner in the Vagrantfile as well

Reply:
I moved the debug work into a separate patch and added provisioners to the vargrant file.